sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -233,6 +310,21 @@ export interface UpdateTagPropertiesOptions extends OperationOptions {
     canWrite?: boolean;
 }
 
+// @public
+export interface UploadBlobResult {
+    digest: string;
+}
+
+// @public
+export interface UploadManifestOptions extends OperationOptions {
+    tag: string;
```

Comment:
this is some service concept of a tag and not an etag, right?

Reply:
Yes, that's right -- a tag is a named reference to an artifact in the repository that can be used instead of a digest. It's a similar concept to tags in Git. For example, the `latest` tag might point to the latest version of an image in the repository.

The first section of [this article](https://blog.atomist.com/docker-image-tags/) gives a decent explanation.